packages/connectivity/src/scp-cf/token-accessor.spec.ts

```diff
@@ -58,7 +59,7 @@ describe('token accessor', () => {
         iss: 'https://testeroni.example.com'
       });
       mockClientCredentialsGrantCall(
-        'https://testeroni.example.com',
+        `https://testeroni.${uaaDomain}`,
```

Comment:
There is a slight difference in how the lib builds the [token url.](https://github.wdf.sap.corp/CPSecurity/node-xs2sec/blob/07a3b01159e82a4207fd50250e83571a353a1782/src/service/XsuaaService.js#L202) 
Th